### PR TITLE
Codify marking unknown subjects without LCSH IDs

### PR DESCRIPTION
**Pull request by bentley**
I don’t know RestructuredText so this formatting might need improvement.

### CODE REVIEW

**Comment by acabal** [comment]
The language in the manual is declarative, in other words it's not "If this, then do that" rather it's "this is how it is." So we should reword this to sound something like "Subject headings that do not have IDs in the LoC system are set to `Unknown`".

**Comment by bentley** [comment]
Like this?

**Comment by acabal** [comment]
Great, thanks!
